### PR TITLE
chore(home): moves shows module up

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

**Comment by damassi** [comment]
@dzucconi  - mind adding a screenshot of logged in view? Noticing over here that there's 
- Recently Viewed
- Similar to ...
- Recently Saved 
- Similar to ... 

And those user-centric items seem more important than shows being second.

**Comment by dzucconi** [comment]
That is the logged in view

**Comment by dzucconi** [comment]
OK so I'll have to get crafty about threading this in.

**Comment by damassi** [comment]
Can we move it below the user-centric items? Or are you logged in from a fresh account and so those rails don't appear? 

The main idea is to pull this shows content up above the sea of gene rails, which are numerous and somewhat arbitrary.

**Comment by damassi** [comment]
> OK so I'll have to get crafty about threading this in.

(Don't want to block things -- only if its straightforward)

**Comment by dzucconi** [comment]
@damassi take a look at aa1cb6ca794f16751b5f22400719c86754dbbca8 — not pretty but it does the trick.

### Logged in:
https://user-images.githubusercontent.com/112297/125680766-4cc86829-1db1-4ff8-bec7-93675b6fc570.mov

### Logged out:
https://user-images.githubusercontent.com/112297/125683497-360644fd-b598-4f73-a531-a013ce2996cf.mov

File: src/v2/Apps/Home/Components/HomeArtworkModules.tsx [hunk 1]
```diff
@@ -16,12 +18,50 @@ const HomeArtworkModules: React.FC<HomeArtworkModulesProps> = ({
...
+              relatedArtistID: "IGNORE",
+              followedArtistID: "IGNORE",
+            },
+          },
+          artworkModule,
+        ]
+      }
+
+      return artworkModule
+    })
```
Comment (damassi) [inline src/v2/Apps/Home/Components/HomeArtworkModules.tsx:47]: 🚢 🚀

**Review by damassi — APPROVED** [review]
💯

**Pull request merged**